sdk/personalizer/test-resources.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+		"baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+		},
+		"location": {
+            "type": "string",
+            "defaultValue": "eastus2euap",
```

Comment:
I recommend not defaulting to an euap region, unless you want the tests to always run in this region (by default this will get overridden in the pipeline to westus)

Reply:
Yes, we prefer to run the tests in the eaup region since we use that as a canary for our service. So I feel it is best to exercise the tests in that region as well so we can catch issues early on - should there be any regressions.

Reply:
Fixed. Updated tests.yml and removed the region override in test-resources.json.